cves/kernel/CVE-2017-16913.yml

```diff
@@ -34,11 +38,11 @@ announced_instructions: |
   This is not the same as published date in the NVD - that is below.
 
   Please enter your date in YYYY-MM-DD format.
-announced_date: '2018-01-31'
+announced_date: 2017-12-07
 published_instructions: |
   Is there a published fix or patch date for this vulnerability?
   Please enter your date in YYYY-MM-DD format.
-published_date: '2018-01-31'
+published_date: 2017-02-13 # https://lore.kernel.org/stable/151853943618565@kroah.com/T/
```

Comment:
Again, comment is unnecessary.

Reply:
Read previous comment - https://github.com/VulnerabilityHistoryProject/vulnerabilities/pull/209#discussion_r1390334647